src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -126,6 +126,28 @@ public function findForDetailsPage($id)
     /**
      * {@inheritdoc}
      */
+    public function findOneForPayment($id)
+    {
+        $queryBuilder = $this->createQueryBuilder('o');
```

Comment:
Yes, it is needed, because: https://github.com/Sylius/Sylius/pull/5321/files#diff-b3c4fa070df7e585e4d320386f45b379R138